addons/website_event/static/src/snippets/s_live_events/000.xml

```diff
@@ -0,0 +1,31 @@
+<templates>
+    <t t-name="website_event.s_live_event_template">
+        <t t-if="event">
+            <div class="o_not_editable">
+                <div class="d-flex justify-content-center m-2">
+                    <div class="d-flex align-items-center w-75 w-sm-50">
+                        <h4 class="text-white bg-danger rounded px-3 py-1 m-2 text-nowrap">
+                            <i class="fa fa-circle me-2"/>LIVE
+                        </h4>
+                        <h2 class="mt-2 text-truncate" style="max-width: 800px;">
+                            <t t-out="event.name"/>
+                        </h2>
+                    </div>
+                </div>
+                <t t-set="background_image" t-value="event.cover_properties.split('background-image')[1].split('url(')[1].split(')')[0].replaceAll('\'', '')"/>
```

Comment:
cover_properties is supposed to be a valid json (even though it's stored as text for some reason..), you should use `JSON.parse`, and if it fails maybe show a fallback.

And then use t-att-style with background-image so we don't have to do manipulations ideally.